music_assistant/providers/audible_audiobooks/audible_helper.py

```diff
@@ -0,0 +1,334 @@
+"""Helper for parsing and using audible api."""
+
+import hashlib
+import html
+import json
+import re
+from typing import Any
+from urllib.parse import parse_qs, urlparse
+
+import audible
+import audible.register
+from audible import AsyncClient
+from music_assistant_models.enums import (
+    ContentType,
+    ImageType,
+    MediaType,
+    StreamType,
+)
+from music_assistant_models.errors import LoginFailed
+from music_assistant_models.media_items import (
+    Audiobook,
+    AudioFormat,
+    MediaItemChapter,
+    MediaItemImage,
+    ProviderMapping,
+    UniqueList,
+)
+from music_assistant_models.streamdetails import StreamDetails
+
+from music_assistant.mass import MusicAssistant
+
+CACHE_DOMAIN = "audible"
+CACHE_CATEGORY_API = "api"
+CACHE_CATEGORY_AUDIOBOOK = "audiobook"
+CACHE_CATEGORY_CHAPTERS = "chapters"
+
+
+class AudibleHelper:
+    """Helper for parsing and using audible api."""
+
+    mass: MusicAssistant
```

Comment:
I don't think these are needed, mypy should be able to work it out from __init__.